### PR TITLE
 Filtering out Lost in HMIS for sites not reported

### PR DESCRIPTION
**Pull request by Marymary-dev**

### CODE REVIEW

File: Scripts/ODS/load_intermediate_tables/CT_intermediate_tables/4_load_intermediate_ARTOutcomes.sql [hunk 1]
```diff
@@ -43,6 +50,12 @@ BEGIN
 		LastPatientEncounter.NextAppointmentDate,
 		CASE
             When  Latestexits.ExitReason  in ('DIED','dead','Death','Died') THEN 'D'--1
+WHEN DATEDIFF(day, ISNULL(LastPatientEncounter.NextAppointmentDate, ART.ExpectedReturn), EOMONTH(DATEADD(month, -1, GETDATE()))) > 30
+    AND LatestExits.ExitReason IS NULL
+    AND DateUploaded Not BETWEEN DATEADD(day, 1 - DAY(GETDATE()), GETDATE())  -- First day of current month
+    AND DATEADD(day, 15 - DAY(GETDATE()), GETDATE()) -- 15th day of current month
+THEN 'LostinHMIS'
```
Comment (nobert-mumo) [inline Scripts/ODS/load_intermediate_tables/CT_intermediate_tables/4_load_intermediate_ARTOutcomes.sql:57]: @Marymary-dev I was wondering is it expected all facilities upload data between 1st and 15th or some can upload in the last week of the month ?

I think we need to also update the Dim ARTOutcome with the new status

**Comment by Marymary-dev** [comment]
All facilities should upload data between 1st and 15th and not the last day of the previous month

File: Scripts/ODS/load_intermediate_tables/CT_intermediate_tables/4_load_intermediate_ARTOutcomes.sql (hunk 1 above)
Comment (Marymary-dev) [reply Scripts/ODS/load_intermediate_tables/CT_intermediate_tables/4_load_intermediate_ARTOutcomes.sql]: @nobert-mumo  requesting a re-review

File: Scripts/ODS/load_intermediate_tables/CT_intermediate_tables/4_load_intermediate_ARTOutcomes.sql [hunk 2]
```diff
@@ -43,7 +50,16 @@ BEGIN
 		LastPatientEncounter.NextAppointmentDate,
 		CASE
             When  Latestexits.ExitReason  in ('DIED','dead','Death','Died') THEN 'D'--1
+
+WHEN DATEDIFF(day, ISNULL(LastPatientEncounter.NextAppointmentDate, ART.ExpectedReturn), EOMONTH(DATEADD(month, -1, GETDATE()))) > 30
+    AND LatestExits.ExitReason IS NULL
+    AND DateUploaded Not BETWEEN DATEADD(day, 1 - DAY(GETDATE()), GETDATE())  -- First day of current month
+    AND DATEADD(day, 20 - DAY(GETDATE()), GETDATE()) -- 20th day of current month
+THEN 'Lost in HMIS'
```
Comment (nobert-mumo) [inline Scripts/ODS/load_intermediate_tables/CT_intermediate_tables/4_load_intermediate_ARTOutcomes.sql:58]: @Marymary-dev I believe this should be `LHMIS` and not `Lost in HMIS` in order for it to be joined on the Dim ARTOutcome ?

File: Scripts/NDWH/ALL DIMENSIONS/STATIC DIMENSIONS/load_DimARTOutcome.sql [hunk 3]
```diff
@@ -20,6 +20,8 @@ BEGIN
 		select'uL' as ARTOutcome
```
Comment (nobert-mumo) [inline Scripts/NDWH/ALL DIMENSIONS/STATIC DIMENSIONS/load_DimARTOutcome.sql:20]: @Marymary-dev @DennisGibz Not sure if this folder is part of the data pipelines or we need to run the changes manually ?

**Review by nobert-mumo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**